src/views/CVERecord/CVERecord.vue

```diff
@@ -78,8 +93,230 @@ export default {
   data() {
     return {
       NVDBaseUrl: 'https://nvd.nist.gov/view/vuln/detail?vulnId=',
+      usecveRecordStore: usecveRecordStore(),
+      cveId: usecveRecordStore().cveId,
+      showHelpText: false,
+      errorMessage: [],
+      disabled: true,
+      getIdStatusCode: undefined,
     };
   },
+  computed: {
+    isSearching() {
+      return usecveRecordStore().isSearching;
+    },
+    websiteEnv() {
+      return import.meta.env.VITE_WEBSITE_ENVIRONMENT;
+    }
+  },
+  created() {
+    const fullPathArr = this.$route.fullPath.split('id=');
+    if (fullPathArr.length === 2) {
+      const cveId = fullPathArr[1];
+      this.cveId = this.cveIdToUpperCase(cveId);
+      this.validateCveId();
+      if (!this.disabled) {
+        this.startLookup();
+      } else {
+        usecveRecordStore().showHelpText = true;
+      }
+    } else {
+      usecveRecordStore().showHelpText = true;
+    }
+  },
+  watch: {
+    $route(to) {
```

Comment:
Can this use `useRoute()`  like you do in `cveRecordSearchModule.vue`? I think that's more in line with vue3 than `$route`

Reply:
This file is using the Options api, so `$route` is used rather than `useRoute()` which is for the composition API.